docs/_interface/Command-Line-Options.md

```diff
@@ -101,7 +101,7 @@ PERFORMANCE TUNING OPTIONS:
 
 ACCOUNT OPTIONS:
   --unlock value                      Comma separated list of accounts to unlock
-  --password value                    Password file to use for non-interactive password input
+  --password value                    Password file to use for non-interactive password input. Note this will not work for password files formatted with Byte Order Marks etc. as included by default with Windows text files; see https://github.com/ethereum/go-ethereum/issues/19905 for a workaround and updates.
```

Comment:
This is nice, but the change here might we overwritten. I propose we delete this change but keep the FAQ entry.

Reply:
I think it's helpful to have the documentation in a context where people might be likely to look for it; the FAQ is kind of a gunshot approach without as much focus and the answer there  would likely only be found after a short while of frustrated searching.  However, if this content would be overwritten by another source, I propose we change that source.

Reply:
The source for this is `geth --help` output. I don't want to add remarks about unicode BOM there.